.gitignore

```diff
@@ -30,6 +30,7 @@ venv
 *.mmdb
 james.ini
 test-results.xml
+tests/functional/assets
```

Comment:
There's already `tests/functional/assets/style.css` down below - maybe just update that to omit the `style.css`?